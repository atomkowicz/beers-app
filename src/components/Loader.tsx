import * as React from "react";

const Loader = () => {
    return (
        <div className="loader-container">
            <div className="loader">Loading ...</div>
        </div>
    );
};

export default Loader;