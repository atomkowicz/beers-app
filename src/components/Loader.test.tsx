import * as renderer from "react-test-renderer";
import * as React from "react";
import Loader from "./Loader";

test('loader renders correctly', () => {
    const tree = renderer
        .create(<Loader/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});