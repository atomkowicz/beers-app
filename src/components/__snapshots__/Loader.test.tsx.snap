// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  >
    Loading ...
  </div>
</div>
`;

exports[`loader renders correctly 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  >
    Loading ...
  </div>
</div>
`;
